### PR TITLE
First version of Selina's Project

### PR DESCRIPTION
**Pull request by SelinaMaria2**

### CODE REVIEW

**Comment by HIPPIEKICK** [comment]
Hi Selina! Did you forget to add your personal files to the repo? Looks like it's only one HTML element (an `h1`) in your HTML file right now.

**Comment by SelinaMaria2** [comment]
Hi! Ups I've chosen the wrong folder. I hope now its working and I have the correct folder. Thank you!

File: code/about.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Project Name</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+  </head>
+  <body>
+    <header>
+      <menu>
+        <a href="./index.html">Home</a>
+        <a href="./photos.html">Photo Gallery</a>
```
Comment (MichelLaufer) [inline code/about.html:14]: I like that you have included a navbar! 

I would recommend to use the full navbar for each of the HTML-pages, in this case I would also recommend to include About in the navbar and for example to style it in CSS if the user is on that page, for example:
`<a class="active-nav" ...>` and in CSS use a different color and font-weight for .active-nav.

File: code/index.html [hunk 2]
```diff
@@ -7,15 +7,42 @@
...
+      <menu>
+        <a href="./about.html">Über mich</a>
+        <a href="./photos.html">Fotos</a>
+      </menu>
+      <h1>Nadelöhr</h1>
+      <button class="styled-button">zum Atelier</button>
+    </header>
 
-      <img src="example-recipe-background.png" alt="Banana Bread">
+<section class="grid-container">
```
Comment (MichelLaufer) [inline code/index.html:22]: Nice that you use grid! 👍 
I like that you use a wrapping grid-container with grid items.

File: code/index.html [hunk 3]
```diff
@@ -7,15 +7,42 @@
...
+
+  <div class="grid-item1">
+    <h2>Leistungsangebot</h2>
+  </div>
+
+    <div class="grid-item2">
+      <img class ="schneidern"
+      src="images/sewingmachine.jpg" alt="Schneidern">
```
Comment (MichelLaufer) [inline code/index.html:30]: Good that you include alt-tag! Important for accessibility.

I would also suggest to split up the code a little bit to multiple lines, like:
``

File: code/style.css [hunk 4]
```diff
@@ -1,5 +1,129 @@
...
+
+h1 {
+  
+  margin-top: 40px;
+  text-align: center;
+  align-items: center;
+  font-size:90px;
+}
+
+.styled-button {
```
Comment (MichelLaufer) [inline code/style.css:24]: A minor comment here is to delete the empty lines here to make the indentation consistent, for example on line 27 and 29.

File: code/style.css [hunk 5]
```diff
@@ -1,5 +1,129 @@
...
+  grid-template-columns: repeat(4, 1fr);
+  font-size: 30px;
+  text-align: center;
+  grid-gap: 10px;
+  color: black;
+}
+
+.grid-item1 {
+  background-color: transparent;
+}
```
Comment (MichelLaufer) [inline code/style.css:98]: A suggestion to shorten the code below is to use something like:
`.grid-item1, .grid-item2, .grid-item3, .grid-item4 {`
`  background-color: transparent;`
`}`

**Review by MichelLaufer — COMMENTED** [review]
Very good job Selina! Nice that you use CSS grid 👍 

I have made some suggestions below. 

Keep up the good work!

**Pull request closed without merging**